src/Semoda/Semoda/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -18,6 +21,18 @@ public static void AddAppServices(this IServiceCollection collection)
             collection.AddSingleton<MainWindowViewModel>();
             collection.AddSingleton<DashboardPageViewModel>();
             collection.AddSingleton<SettingsPageViewModel>();
+
+            {
+                // Load settings from JSON and add for dependency injection
+                var configuration = new ConfigurationBuilder()
+                   .SetBasePath(Directory.GetCurrentDirectory())
+                   .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                   .Build();
+
+                var appSettings = new AppSettingsModel();
+                configuration.Bind(appSettings);
+                collection.AddSingleton(appSettings);
```

Comment:
Instead uf adding a custom app mode, we should add the IConfiguration to our application. So we are able to seperate the settings in multiple